domain/machine/errors/errors.go

```diff
@@ -6,6 +6,10 @@ package errors
 import "github.com/juju/juju/internal/errors"
 
 const (
+	// MachineDead describes an error that occurs when the machine being
+	// operated on is considered dead.
+	MachineDead = errors.ConstError("machine is dead")
```

Comment:
For units, the error is called `UnitIsDead`

Reply:
Done & Done